session_2/student_registry/db/migrate/20210114070046_create_students.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :roll_number
+      t.string :branch
+      t.decimal :cgpa, precision: 4, scale: 2
```

Comment:
Can we save a student with cgpa `15.00`? That's wrong, right?
Can you override `cgpa=` function (or write validations) to make it work?

The function should take any floating value, throw `ArgumentError` exception if cgpa > 10 or < 0 and store it appropriately.

https://api.rubyonrails.org/classes/ActiveRecord/Base.html#class-ActiveRecord::Base-label-Overwriting+default+accessors

https://guides.rubyonrails.org/active_record_validations.html

Do try your hand at it! :v: